consensus/istanbul/validator/weighted.go

```diff
@@ -617,14 +617,10 @@ func (valSet *weightedCouncil) Policy() istanbul.ProposerPolicy { return valSet.
 //   (1) already has up-do-date proposers
 //   (2) successfully calculated up-do-date proposers
 func (valSet *weightedCouncil) Refresh(hash common.Hash, blockNum uint64, config *params.ChainConfig, isSingle bool, governingNode common.Address, minStaking uint64) error {
+	// TODO-Governance divide the following logic into two parts: proposers update / validators update
```

Comment:
@jeongkyun-oh Can I ask why two logics need to be splitted?

Reply:
I commented for later refactoring. The logic would be nicer if it is divided.

Reply:
Thanks!